thavalon-server/src/game/role.rs

```diff
@@ -87,17 +100,11 @@ impl Role {
     }
 
     pub fn is_lover(self) -> bool {
```

Comment:
Since `Role` implements `Copy`, it has copy semantics instead of move semantics, which is a fancy Rust way of saying we only need to copy the bits rather than deal with ownership or dropping. The same thing happens with integer primitives, which is why this works:
```
let a = 1;
let b = a;
dbg!(a);
```
but it wouldn't if `a` were a `String`.

With small types that implement `Copy`, it's slightly preferred to take `self` instead of `&self` because the compiler can handle it more efficiently: https://rust-lang.github.io/rust-clippy/master/#trivially_copy_pass_by_ref

It's not a huge deal here, but it's a habit I got into a while ago